mypy.ini

```diff
@@ -1,5 +1,4 @@
 [mypy]
-python_version = 3.8
```

Comment:
Is the mypy version now defined somewhere else? I would update the nox check to only run mypy with 3.10, because there have been differences in the past when run with different versions

Reply:
We had issues in the past with mypy giving different results on different python versions, but I would hope this is stabilised now.

Reply:
I think it will work